packages/liveblocks-react/src/umbrella-store.ts

```diff
@@ -1506,9 +1527,7 @@ export class UmbrellaStore<M extends BaseMetadata> {
     batch(() => {
       this.historyVersions.invalidate();
       this.notifications.invalidate();
-      this.optimisticUpdates.invalidate();
       this.permissionHints.invalidate();
-      this.roomNotificationSettings.invalidate();
```

Comment:
We're getting closer to no longer always invalidating everything. These two could now be removed. To get rid of the rest, more (similar) refactoring needs to happen first.